extension/src/experiments/webview/messages.ts

```diff
@@ -424,10 +423,7 @@ export class WebviewMessages {
     )
   }
 
-  private showPlotsToSide() {
-    return commands.executeCommand(
-      RegisteredCommands.EXPERIMENT_AND_PLOTS_SHOW,
-      this.dvcRoot
-    )
+  private showPlots() {
+    return commands.executeCommand(RegisteredCommands.PLOTS_SHOW, this.dvcRoot)
```

Comment:
[F] We had the feedback that opening to the side was too small when performed from a laptop.